automation_tools/__init__.py

```diff
@@ -3133,7 +3159,10 @@ def relink_manifest(manifest_file=None):
     if manifest_file is None:
         manifest_file = download_manifest()
         if os.environ.get('EXP_SUBS_FILE', None) is not None:
-            assert validate_manifest(manifest_file)
+            validate = validate_manifest(manifest_file)
+            assert validate[0]
+            if len(validate) > 1:
+                manifest_file = download_manifest()
```

Comment:
I think that this means we just re-download the manifest if its validation fails. In robottelo-ci for download job we are calling `relink_manifest`:

    https://github.com/SatelliteQE/robottelo-ci/blob/master/scripts/satellite6-manifest-downloader.sh

Shall we rerun validation after second `download_manifest` to be sure all is fine now?

Reply:
Good idea ... let me just recall the validate again .

Reply:
Thank you!